components/tooltip/tooltip.js

```diff
@@ -564,18 +577,30 @@ class Tooltip extends RtlMixin(LitElement) {
 		return this._openDir === 'bottom' || this._openDir === 'top';
 	}
 
+	_isInteractive(ele) {
+		if (ele.nodeType !== Node.ELEMENT_NODE) {
+			return false;
+		}
+		const nodeName = ele.nodeName.toLowerCase();
+		return !!interactiveElements[nodeName];
```

Comment:
Could also use an array with `indexOf(nodeName) > -1`

Reply:
That was how I wrote it originally but I figured I'd use an object for direct key lookup rather than iterating the array (although the perf difference is irrelevant given how small the list is)

Reply:
Yeah likely doesn't matter, and there's `Map` as well but that seems super overkill given the size.